src/objects/chart/chart.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import Vue from 'vue/dist/vue.esm.browser'; //
+import ChartComponent from './chart.vue'; // Our component
+import ChartData from './chart.data.js'; // Our sample data
+
+class Chart {
+  constructor(settings = {}, data = {}) {
+    this.data = data;
+    this.settings = settings;
+  }
+
+  /**
+   * Initializes the module
+   */
+  init() {
+    Vue.component('chart', ChartComponent);
+
+    new Vue({
+      el: '[data-js="app"]',
+      delimiters: ['v{', '}'],
+      data: {
+        'chartData': Chart.data, // set to ChartData in chart.data.js but this can be included anyway you like
+      },
+      methods: {}
+    });
+
+    console.log('Hello World!');
+  }
+
+  /**
+   * Logs constants to the debugger
+   * @param  {object} param - our constants
+   */
+  _constants(param) {
+    console.dir(param);
+  }
+}
+
+Chart.data = ChartData;
```

Comment:
Setting chart data. Another reason to set this here is that it enables you to instantiate multiple versions of this chart app with different sets of data. For example;

```
let chartA = new Chart();
let chartA.data = {'title:'Dataset A', ...}

let chartB = new Chart();
let chartB.data = {'title:'Dataset B', ...}
```
It's not terribly relevant in this case since the Vue component can be reused by passing different props, but for other vanilla js components it makes more sense.